### ISSUE TITLE
Host/Upload assets

### ISSUE THREAD

**Issue by ptz0n** [tictail/gulp-tictail#2]
S3

**Comment by ffdead** [comment]
Any news on getting HTML + Assets uploaded via this script?

Any other recommended workflow instead of copy/pasting markup into the dashboard and manually uploading assets to a third party server?

Tack!

**Comment by kalasjocke** [comment]
@ffdead We're not actively working on adding an automatic way of either hosting or uploading the theme into the Tictail dashboard. Right now you'll need to hoop up some gulp - s3 (or other hosting) workflow and paste the compiled template into the Dashboard manually.

**Comment by ffdead** [comment]
Ok got it - thanks for the quick reply!

Would have been sweet with some kind of watcher similar to https://github.com/Shopify/shopify_theme which uploads local theme files automatically for preview online.

Setting up a third party asset hosting should be fine, mostly worried about copy/pasting and previewing inside the dashboard.

**Comment by kalasjocke** [comment]
@ffdead It would sure have been nice and that was the goal of this project when me and @ptz0n started it way back when. We never came around to prioritize that though. Hopefully you can get some value of previewing your theme locally and then when the time is ready upload it manually to the dashboard.

**Comment by ffdead** [comment]
I didn't feel comfortable using this library since so many tags were missing. Instead, I ended up creating a Bookmarklet tunnel for Livereload that inlines all my local assets and injects them into the theme dashboard for fast preview. It was working quite well until I got this error:
> Your theme can't be saved because it's too large
https://www.dropbox.com/s/l57hbpdjgzrpwqd/Screenshot%202016-02-21%2020.10.18.png?dl=0

I was really hoping to inline them to get fast feedback while developing, and only upload assets to a 3rd party server for the production build.

Anyways, just thought I would keep you updated on my quirky adventures of building a Tictail theme :)

**Comment by kalasjocke** [comment]
Thanks for keeping us up to date on your endeavours. :smile: 

Totally understandable that you moved on @ffdead. If you find the time patches are always welcome to this project.

If you inline images I would argue that that error is valid. We don't want people do develop to large themes really. Can't you just host images on your local machine and upload once your finished?

**Comment by ffdead** [comment]
Yes I am not inlining image assets, only JS and CSS. Turns out I exceeded the size limit when I imported the Javascript `babel-polyfill` - so I'll work around it for now by including it as an external script instead.

Brilliant idea @kalasjocke - loading assets from localhost instead of a 3rd party server while previewing would be lightning fast - will give that a try, thanks!

**Comment by kalasjocke** [comment]
@ffdead Glad I can be of help. Let me know if you encounter any other things that would make developing themes a better experience.
